docs/src/content/en/reference/tools/mcp-client.mdx

```diff
@@ -155,7 +159,9 @@ async disconnect(): Promise<void>
 
 ## Examples
 
-### Basic Usage
+### Static Tool Configuration
```

Comment:
I updated these sections cause I noticed the docs site doesn't show the title unless there's paragraph text too